.github/workflows/test.yml

```diff
@@ -23,17 +23,13 @@ jobs:
         - macos-latest
         - windows-latest
         python-version:
-        - '3.7'
         - '3.8'
```

Comment:
```suggestion
```

Reply:
On top of removing python 3.8 I now also added python 3.12.

Reply:
Makes sense, thanks for the explanation!